src/Network/Network/Models/PSPrivateEndpointConnection.cs

```diff
@@ -32,6 +32,10 @@ namespace Microsoft.Azure.Commands.Network.Models
 {
     public partial class PSPrivateEndpointConnection : PSChildResource
     {
+
+        [JsonProperty(Order = 1)]
+        [Ps1Xml(Target = ViewControl.Table)]
+        public string ProvisioningState { get; set; }
```

Comment:
So, we should use unique numbers in the property order

Reply:
Thanks, we removed the attribute.